packages/app/src/service-container.ts

```diff
@@ -96,6 +106,9 @@ export class ServiceContainer {
         if (!this.appModule.isProvided(Logger)) {
             this.appModule.addProvider({ provide: Logger, useFactory: (t: ConsoleTransport) => new Logger([t]) });
         }
+        if (!this.appModule.isProvided<LoggerInterface>()) {
+            this.appModule.addProvider(provide<LoggerInterface>({ useExisting: Logger }));
+        }
```

Comment:
Perhaps we should rename `LoggerInterface` to `Logger` and `Logger` to `LoggerImpl`?

Reply:
this should be fixed in master without this change, since we track now interfaces in the type itself so that provided `Logger` is matched for deps `LoggerInterface`

Reply:
@marcj do you want me to revert this part then?